src/modules/person/api/person.controller.ts

```diff
@@ -61,9 +62,19 @@ export class PersonController {
     @ApiUnauthorizedResponse({ description: 'Not authorized to create the person.' })
     @ApiForbiddenResponse({ description: 'Insufficient permissions to create the person.' })
     @ApiInternalServerErrorResponse({ description: 'Internal server error while creating the person.' })
-    public async createPerson(@Body() params: CreatePersonBodyParams): Promise<void> {
+    public async createPerson(@Body() params: CreatePersonBodyParams): Promise<PersonendatensatzResponse> {
```

Comment:
I think the response according to SchulConneX is just the Person and not the Personendatensatz.

Reply:
The http response status code must be 200 und not 201 if the create was successful.